scripts/run.js

```diff
@@ -17,7 +17,7 @@ const main = async () => {
     console.log("Contract deployed to:", slotMachineContract.address);
     console.log("Owner of the contract:", owner.address);
 
-    const randNums = await slotMachineContract.logicOfLuck("0.23");
+    const randNums = await slotMachineContract.play(web3.utils.toWei("0.1", 'ether'));
```

Comment:
why is this converted to wei? If we are doing the conversion in the smart contract?

Reply:
The conversion happens in the web app (front end) using web3.js. For local testing (run.js) need to provide same input of WEI.